js/containers/portal-dashboard/portal-dashboard-app.tsx

```diff
@@ -229,6 +237,10 @@ class PortalDashboardApp extends React.PureComponent<IProps, IState> {
     this.setState({ viewMode: mode });
   }
 
+  private setListViewMode = (value: boolean) => {
```

Comment:
Here's the origin of the function.  This should probably be something like `toggleQuestionMode`.  Since you're not setting a mode, you're toggling a boolean.  I would mass change the instances of `setListViewMode` to something more in line with what the function does (like `toggleQuestionMode`).

Reply:
Per my comment below, I think the best approach is to change this to:
`private setListViewMode = (value: ListViewMode) => {`